code/modules/atmospherics/machinery/components/tank.dm

```diff
@@ -0,0 +1,521 @@
+/obj/machinery/atmospherics/components/tank
```

Comment:
watch this in particular as it was changed with tg's systems in mind, verified everything worked but incase of wonkyness inside the tanks check the gas procs first